src/main/webapp/WEB-INF/static/js/trialmanager/subobservations/subObservationSet.js

```diff
@@ -1695,12 +1695,21 @@
 					columnData: '=',
 					isCategoricalDescriptionView: '='
 				},
-				controller: function ($scope) {
+				controller: function ($scope, BREEDING_METHOD_SCALE) {
+					$scope.targetkey = 'targetkey';
+					$scope.valuecontainer = {targetkey : $scope.observation.value};
```

Comment:
I think this is more appropriate:

$scope.targetkey = 'observationValue';
$scope.valuecontainer = {observationValue : $scope.observation.value};